src/vs/workbench/contrib/chat/browser/chatWidget.ts

```diff
@@ -351,6 +354,7 @@ export class ChatWidget extends Disposable implements IChatWidget {
 		}));
 		this._register(this.tree.onDidFocus(() => {
 			this._onDidFocus.fire();
+			this.chatListFocused.set(this.tree.isDOMFocused());
```

Comment:
Does this need to be updated when `this.tree.onDidBlur` fires?

Reply:
_technically_ it's fine, but it looks like it should, so I'll just add it